core/src/main/kotlin/org/evomaster/core/problem/graphql/GraphqlCallResult.kt

```diff
@@ -1,2 +1,71 @@
-package org.evomaster.core.problem.graphql
+package org.evomaster.core.problem.grapqhl
 
+import com.google.gson.Gson
+import com.google.gson.JsonObject
+import org.evomaster.core.search.ActionResult
+
+class GraphqlCallResult : ActionResult {
```

Comment:
this looks like a copy&paste of `RestCallResult`. in such case, should rather extend it to avoid code duplication